stock_valuation_mrp/models/product_product.py

```diff
@@ -0,0 +1,52 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields, api
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    has_waste_cost_mgmt = fields.Boolean(
+        string='Has cost management residue',
+        default=False,
+        tracking=True,
+        readonly=False,
+        compute='_compute_has_waste_cost_mgmt',
+        store=True,
+        help="""
+        When enabled, product residue will be covered with cost management
+        """,
+    )
+
+    residue_pricelist_mgmt = fields.Many2one(
+        comodel_name='product.pricelist',
+        string='Pricelist for management residue',
+        default=False,
```

Comment:
This is unnecessary (it's relevant for `Boolean` fields, but for `Many2one` field is obviously the default value...)